Documentation/English/Supported.md

```diff
@@ -0,0 +1,194 @@
+
+# Supported
+
+*Configurations of hardware & software that you can use.*
+
+<br>
+
+## Games
+
+<br>
+
+### **[Illusion]**
+
+<br>
+
+[<img
```

Comment:
I like the table approach better because it is more readable in plain text, more informative, screen reader friendly, and easier to maintain.
The logos look nice, but I don't think there is a place to incorporate them at the moment.

Reply:
Well for screen readers I could just add `alt` attributes~

With the github pages approach that I suggested above, I could also create a simple auto-generation tool for stuff like that~

Reply:
I do think it's more decorative, I just don't see any practical use for it as compared to the current version.